patterns/2-structured/base-documentation.md

```diff
@@ -133,6 +133,7 @@ started right away: [README-template.md](templates/README-template.md),
 * Mercado Libre - create a documentation site that contains how to get started with InnerSource and also define the basic artifacts that a repository must have to be InnerSource (README, CONTRIBUTING, CODING_GUIDELINES, etc).
 * Analog Devices Inc.
 * Airbus
+* Siemens automatically creates a checklist issue for every new InnerSource project to make maintainers aware of mandatory requirements as well as best practices.
```

Comment:
Very cool!

Can you share a bit more about what your mandatory requirements are for InnerSource projects?

Also how specific are you about this? e.g. do you just say "please create a CONTRIBUTING.md" or do you say "create a CONTRIBUTING.md that contains at least the information X, Y, and Z"?

Reply:
We cannot share all the details, but maybe this would be helpful?

```suggestion
* Siemens automatically creates a checklist issue for every new InnerSource project to make maintainers aware of mandatory requirements (e.g. business approval, license & copyright, export control, contributing guidelines, maintainer maturity) as well as best practices (e.g. documentation as code, semantic versioning, continuous integration/deployment).
```